site/access-control.xml

```diff
@@ -297,18 +337,45 @@ limitations under the License.
         This will check LDAP first, and then fall back to the internal
         database if the user cannot be authenticated through LDAP:
       </p>
+
+<pre class="example">
+auth_backends.1 = ldap
+auth_backends.2 = internal
+</pre>
+
+          Or, in the <a href="/configure.html">classic config format</a>:
+
       <pre class="example">
 [{rabbit, [
             {auth_backends, [rabbit_auth_backend_ldap, rabbit_auth_backend_internal]}
           ]
 }].
       </pre>
-      
+
       <p>
         Same as above but will fall back to the <a
         href="https://github.com/rabbitmq/rabbitmq-auth-backend-http">HTTP backend</a>
         instead:
       </p>
+
+<pre class="example">
+# rabbitmq.conf
+#
+auth_backends.1 = ldap
+# uses module name instead of a short alias, "http"
+auth_backends.2 = rabbit_auth_backend_http
+</pre>
+<pre class="example">
+%% advanced.config
+%% See HTTP backend docs for details
+{rabbitmq_auth_backend_http,
```

Comment:
`http.user_path`
`http.vhost_path`
`http.resource_path`
are defined in http backend plugin schema https://github.com/rabbitmq/rabbitmq-auth-backend-http/commit/571937f034fd4829ca2dfdb732de87df387c43f1